From 050127548d7f28464cbe5dc9caa83c8023676d80 Mon Sep 17 00:00:00 2001
From: Chirag Shilwant <c-shilwant@ti.com>
Date: Thu, 13 Feb 2025 14:33:50 +0530
Subject: [PATCH 1/3] lite: Update CMakeLists for building
 _pywrap_tensorflow_interpreter_wrapper

- Update the CMakeLists.txt file for TensorFlow Lite to include the paths for
PYTHON, Numpy, and PYBIND11 headers.

- This ensures that these headers are available during the build process,
which is necessary for building the _pywrap_tensorflow_interpreter_wrapper
target that relies on these dependencies.

Upstream-Status: Inappropriate [configuration]

Signed-off-by: Chirag Shilwant <c-shilwant@ti.com>
---
 tensorflow/lite/CMakeLists.txt | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/tensorflow/lite/CMakeLists.txt b/tensorflow/lite/CMakeLists.txt
index bce9627fbd3..b7bb39869b5 100644
--- a/tensorflow/lite/CMakeLists.txt
+++ b/tensorflow/lite/CMakeLists.txt
@@ -810,6 +810,9 @@ set_target_properties(_pywrap_tensorflow_interpreter_wrapper PROPERTIES PREFIX "
 target_include_directories(_pywrap_tensorflow_interpreter_wrapper
   PUBLIC
     ${TENSORFLOW_SOURCE_DIR}
+    ${PYTHON_TARGET_INCLUDE}
+    ${NUMPY_TARGET_INCLUDE}
+    ${PYBIND11_TARGET_INCLUDE}
 )

 target_link_libraries(_pywrap_tensorflow_interpreter_wrapper
--
2.34.1
